distribution/changelog.txt

```diff
@@ -110,6 +110,7 @@
 - Fix: [#6904] Manually added multiplayer servers not saved.
 - Fix: [#7003] Building sloped paths through flat paths with clearance checks off causes glitches.
 - Fix: [#7011] Swinging and bobsleigh cars going backwards swing in the wrong direction (original bug).
+- Fix: [#7042, #7077] Paths sometimes disconnect when building them with clearance checks off.
```

Comment:
Did these issues occur in v0.1.1? I was under the impression it was a recent regression.

Reply:
@AaronVanGeffen I'm pretty sure they did. These issues have been around for a very long time.

Reply:
It's been around since disabled clearance was introduced

Reply:
It's been around since disabled clearance was introduced